src/core/proj/qgscoordinatereferencesystem.cpp

```diff
@@ -2714,11 +2799,12 @@ int QgsCoordinateReferenceSystem::syncDatabase()
                 .arg( QgsSqliteUtils::quotedString( authority ) )
                 .arg( QgsSqliteUtils::quotedString( code ) )
                 .arg( isGeographic ? 1 : 0 )
-                .arg( deprecated ? 1 : 0 );
+                .arg( deprecated ? 1 : 0 )
+                .arg( QgsSqliteUtils::quotedString( srsTypeString ) );
         }
         else
         {
-          sql = QStringLiteral( "INSERT INTO tbl_srs(description,projection_acronym,ellipsoid_acronym,parameters,srid,auth_name,auth_id,is_geo,deprecated) VALUES (%2,%3,%4,%5,%6,%7,%8,%9,%10)" )
+          sql = QStringLiteral( "INSERT INTO tbl_srs(description,projection_acronym,ellipsoid_acronym,parameters,srid,auth_name,auth_id,is_geo,deprecated,srs_type) VALUES (%2,%3,%4,%5,%6,%7,%8,%9,%10,%11)" )
```

Comment:
%1 is missing here (not a change you made but a preexisting condition), should we fix?